tools/scripts/used-headers.txt

```diff
@@ -24,6 +24,7 @@ sys/wait.h
 unistd.h
 # iceoryx_posh / iceoryx_hoofs headers
 algorithm
+array
```

Comment:
Would it be much work to revert this change?

Reply:
Of course not, but is it necessary? It is currently needed to pass the checks when using `std::array` as substitute for a C array.

It all depends on whether we want/can use `std::array` which I am in favor of.